src/flownet/templates/observations.yamlobs.jinja2

```diff
@@ -1,10 +1,13 @@
 smry:
  {%- for well_name in schedule.get_wells(kw_class="WCONHIST") -%}
    {%- if (error_config.WOPR.rel_error is not none) and (error_config.WOPR.min_error is not none) -%}
-   {%- for kw in schedule.get_keywords(kw_class="WCONHIST", well_name=well_name, ignore_nan="oil_rate") -%}
+   {%- for kw in schedule.get_keywords(kw_class="WCONHIST", well_name=well_name, ignore_nan="oil_rate", date=schedule.get_dates()[num_beginning_date:num_end_date]) -%}
    {% if loop.first and not loop.last: %}
  - key: WOPR:{{ well_name }}
    observations:
+    - date: {{ kw.date.strftime('%Y-%m-%d') }}
+      value: {{ kw.oil_rate }}
+      error: {{ [error_config.WOPR.rel_error * kw.oil_rate, error_config.WOPR.min_error] | max }}
```

Comment:
```suggestion
    - date: {{ kw.date.strftime('%Y-%m-%d') }}
      value: {{ kw.oil_rate }}
      error: {{ [error_config.WOPR.rel_error * kw.oil_rate, error_config.WOPR.min_error] | max }}
```

Isn't this double (see line below). I think you could end the if statement just behind the `observations:`. Then you don't have to write the date/value/error twice the same.

This exact same reduction can potentially be applied 9 times which saves quite a few lines of code.